### PR TITLE
Process info tome

### LINKED ISSUE #490: Add a process info tome
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**
Add a tome to get process info based on PID or current process if -1 is specified.

**Describe alternatives you've considered**
None

**Additional context**
None

### PR DESCRIPTION
**Pull request by Alextibtab**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Added tome for getting process information or the current process if -1 is specified. I tried a few different formats for the output think this one looks okay but let me know if you think it needs tweaking. Also the eldritch documentation indicates that the "cmd" and "environ" keys in the dict should be list/arrays but currently they're just one big string.

#### Which issue(s) this PR fixes:

Fixes #490

### CODE REVIEW

File: tavern/tomes/process_info/main.eldritch [hunk 1]
```diff
@@ -0,0 +1,17 @@
+def process_info(pid):
+    info = process.info(sys.get_pid() if pid == -1 else pid)
```
Comment (hulto) [inline tavern/tomes/process_info/main.eldritch:2]: Nice 😎

**Review by hulto — COMMENTED** [review]
I'll test when I get home but looks very promising

File: tavern/tomes/process_info/main.eldritch [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        if key == "cmd":
+            print("cmd_args:\n")
+            for nested_value in value.split(" "):
+                print("\t- {}\n".format(nested_value))
+        elif key == "environ":
+             print("env_variables:\n")
+             for nested_value in value.split(","):
+                print("\t- {}\n".format(nested_value))
+        else:
+            print("{}={}\n".format(key, value))
```
Comment (hulto) [inline tavern/tomes/process_info/main.eldritch:13]: Nit could you change the `=` to `: ` for consistency

```
pid=8531
name=golem
cmd_args:
        - /workspaces/realm/implants/target/debug/golem
        - /tmp/test.eldritch
exe=/workspaces/realm/implants/target/debug/golem
env_variables:
        - BROWSER=/vscode/vscode-server/bin/linux-x64/8b3775030ed1a69b13e4f4c628c612102e30a681/bin/helpers/browser.sh
        - CARGO=/usr/local/rustup/toolchains/nightly-2023-09-04-x86_64-unknown-linux-gnu/bin/cargo
        - CARGO_HOME=/usr/local/cargo
        - CARGO_MANIFEST_DIR=/workspaces/realm/implants/golem
        - CARGO_PKG_AUTHORS=
 ```

**Review by hulto — APPROVED** [review]
One nit otherwise LGTM.

File: tavern/tomes/process_info/main.eldritch (hunk 2 above)
Comment (Alextibtab) [reply tavern/tomes/process_info/main.eldritch:13]: should be fixed

**Review by hulto — APPROVED** [review]

**Pull request merged**